### ISSUE TITLE
[REQUEST] Download save IMG to folder

### ISSUE THREAD

**Issue by jackmaokay** [openbullet/OpenBullet2#690]
I'm using OpenBullet 2 version of WebClient
I want to download the images using the URL that has been parsing
about the folder I want, but I can't do it because I don't know C#
Do you have any solution?
IMG => [https://ibb.co/WVh8HP2](https://ibb.co/WVh8HP2)

**Comment by ewwink** [comment]
use Utility > FileWriteBytes

```python
BLOCK:HttpRequest
  url = "https://avatars.githubusercontent.com/u/1999"
  TYPE:STANDARD
  $""
  ""
ENDBLOCK

BLOCK:CreatePath
  path = "myImages"
ENDBLOCK

BLOCK:FileWriteBytes
  path = "myImages\\test.jpg"
  content = @data.RAWSOURCE
ENDBLOCK
```

**Comment by jackmaokay** [comment]
> use Utility > FileWriteBytes

thank you
I was done with this before I got an answer from you, but thank you again anyway for taking care of this matter.
I have one more question to ask.
I want to create a for loop
The word Parse is retrieved by Recursive with value = [list,forlist,newlist,oldlist,savelist]
now how can i put it in the BLOCK HTTP REQUEST URL ?
for example URL=link/<Parsevalue>/file.html
that it can run all the Parse that I took out earlier?
